project_setup/scripts/terraform-to-secrets

```diff
@@ -156,6 +156,26 @@ def parse_tagged_outputs(
     return
 
 
+def find_resources_in_child_modules(
+    child_modules: list, resource_type: Optional[str]
+) -> Generator[Dict, None, None]:
+    """
+    Search for resources of a certain type in a Terraform child_modules list.
+
+    resource_type None yields all resources.
+    """
+    for child_module in child_modules:
+        for resource in child_module["resources"]:
```

Comment:
Is there ever a case where `child_module` won't have a `resources` key? Currently if that occurs, this looks it'll break everything instead of returning an empty list or other desired failure state.

Reply:
I don't think we've seen a case where that can happen, but it never hurts to make our code more robust - see https://github.com/cisagov/development-guide/pull/20/commits/6b2f5f8ed6832bdd99fe14e67d78ddb51aeeb5dd.

Thanks for the suggestion!